src/Backend/Modules/ContentBlocks/Installer/Data/install.sql

```diff
@@ -3,7 +3,7 @@ CREATE TABLE IF NOT EXISTS `content_blocks` (
  `revision_id` int(11) NOT NULL auto_increment,
  `user_id` int(11) NOT NULL,
  `extra_id` int(11) NOT NULL,
- `template` varchar(255) NOT NULL default 'default.tpl',
+ `template` varchar(255) NOT NULL default 'Default.html.twig',
```

Comment:
it has been https://github.com/forkcms/forkcms/pull/1566/files#diff-32212ef9f7bc7b6d110619bd2798369bR174